server/dns.go

```diff
@@ -3,6 +3,7 @@ package server
 
 import (
 	"fmt"
+	sham "github.com/nanopack/shaman/core/common"
```

Comment:
I know this is pedantic, but would you mind moving this to the bottom of the import block (under `"github.com/nanopack/shaman/core"`), thanks.

Reply:
sure